dlt/extract/extractors.py

```diff
@@ -63,6 +66,7 @@ def __init__(
         self._filtered_columns: Dict[str, Dict[str, TSchemaEvolutionMode]] = {}
         self._storage = storage
         self._caps = _caps or DestinationCapabilitiesContext.generic_capabilities()
+        self._plugins = Container()[PluginsContext].plugins
```

Comment:
it would be nicer to have a decorator similar to with_config to inject the plugins here.